platform/backend/src/routes/mcp-server-installation-requests.ts

```diff
@@ -450,16 +448,10 @@ const mcpServerInstallationRequestRoutes: FastifyPluginAsyncZod = async (
           });
         }
 
-        // Get user name from database
-        const [userData] = await db
-          .select()
-          .from(schema.usersTable)
-          .where(eq(schema.usersTable.id, user.id));
-
         const updatedRequest = await McpServerInstallationRequestModel.addNote(
           id,
           user.id,
-          userData.name,
+          user.name,
```

Comment:
`request.user` was added as part of RBAC work -- no need to query _again_ for the user, we already have `user.name`